Makefile

```diff
@@ -32,8 +32,13 @@ install: ## Runs the install script and vendors golang dependencies
 goclient: ## Runs the old golang survey client to play cribbage
 	go run localclient/main/main.go
 
+tailwind: client/src/styles.css
+
+client/src/styles.css: client/src/tailwind.css
+	cd client/ && npx tailwindcss build src/tailwind.css -o src/styles.css
```

Comment:
Do you have documentation on why we built this this way?

The way `cribbage` uses a Makefile is not how Makefiles were intended to be used; so I'd prefer sticking with our "agreed upon" divergence from recommendations which will look different than its currently written. If the only need is to build the output `src/styles.css`, I'd suggest doing:

```Makefile
.PHONY: tailwind
tailwind:
	cd client/ && npm tailwindcss build src/tailwind.css -o src/styles.css
```

Reply:
The above rule is what I originally had. The point of doing it the way I have it now was so the css doesn't get regenerated unless `tailwind.css` changes. Is it so bad to use a makefile how it was intended in this instance? It's not too time consuming to rebuild every time, so I suppose I'd be okay with using a phony target

Reply:
That sounds good. Could we add a help command to the targets as written? Running `make help` should print out a docstring for each target